src/components/Sidebar.vue

```diff
@@ -2,22 +2,29 @@
   .sidebar-container
     .sidebar-upper-section
       .sidebar-header
-        h2.sidebar-title.text-big
-          | Frontend Cookbook
+        h2.sidebar-title.sidebar-title-wolox
+          | wolox
```

Comment:
Is lowercase OK?

Reply:
Yes, because with CSS i transform it to uppercase